### PR TITLE
Add a management command to repudiate (force expire) ActivityLogTokens

### LINKED ISSUE #3556: Add a way of manually repudiating tokens
Splitting out #3349 to make it easier to track

### PR DESCRIPTION
**Pull request by eviljeff**
Fixes #3556

### CODE REVIEW

File: src/olympia/activity/management/commands/repudiate_token.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+                uuid__in=args)]
+            log.info(u'%s tokens (%s) expired' % (len(done), ','.join(args)))
+            if version_pk:
+                print 'Warning: --version_id ignored as tokens provided too'
+        elif version_pk:
+            done = [t.expire() for t in ActivityLogToken.objects.filter(
+                version__pk=version_pk)]
+            log.info(
+                u'%s tokens for version %s expired' % (len(done), version_pk))
+        else:
```
Comment (diox) [inline src/olympia/activity/management/commands/repudiate_token.py]: This branch is not tested

File: src/olympia/activity/management/commands/repudiate_token.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+                    help='Expire all tokens on this version.'),
+    )
+
+    def handle(self, *args, **options):
+        version_pk = options.get('version_id')
+        if len(args) > 0:
+            done = [t.expire() for t in ActivityLogToken.objects.filter(
+                uuid__in=args)]
+            log.info(u'%s tokens (%s) expired' % (len(done), ','.join(args)))
+            if version_pk:
```
Comment (diox) [inline src/olympia/activity/management/commands/repudiate_token.py]: This branch is not tested

Comment (eviljeff) [inline src/olympia/activity/management/commands/repudiate_token.py]: Added in https://github.com/mozilla/addons-server/pull/3564/commits/fbc75bc6f61d940ee49716018da31c91d82ce8af

File: src/olympia/activity/management/commands/repudiate_token.py (hunk 1 above)
Comment (eviljeff) [inline src/olympia/activity/management/commands/repudiate_token.py]: Added in https://github.com/mozilla/addons-server/pull/3564/commits/fbc75bc6f61d940ee49716018da31c91d82ce8af

**Pull request merged**